scrape/target.go

```diff
@@ -478,6 +478,11 @@ func populateLabels(lset labels.Labels, cfg *config.ScrapeConfig) (res, orig lab
 	return res, preRelabelLabels, nil
 }
 
+// Delegate targetsFromGroup
+func TargetsFromGroup(tg *targetgroup.Group, cfg *config.ScrapeConfig) ([]*Target, []error) {
+	return targetsFromGroup(tg, cfg)
+}
```

Comment:
I think it would be fine to just make the existing function public and change the calls, it only shows up in like 2 places.